src/ildasm/dis.cpp

```diff
@@ -1646,7 +1646,7 @@ BOOL Disassemble(IMDInternalImport *pImport, BYTE *ILHeader, void *GUICookie, md
 
             case ShortInlineBrTarget:
             {
-                char offset = (char) pCode[PC];
+                signed char offset = (signed char) pCode[PC];
```

Comment:
> However I prefer modifying C/C++ source code to signed char if we need signed char, because C/C++ doesn't define char is sign or unsigned.

This is ported code like other pieces with the assumption of MS compiler. We cannot simply find all such instances. Our porting strategy is to lay PAL or equivalent operation with MS compiler in case there is difference like this.